### PR TITLE
Chart issues

### PR DESCRIPTION
**Pull request by shayanbinary**

### CODE REVIEW

File: src/components/Dialog.jsx [hunk 1]
```diff
@@ -9,13 +9,20 @@ class Dialog extends Component {
             children,
             onContainerClick,
             className,
+            isMobile
         } = this.props;
 
+        const inner_style = ( isMobile && open) ? {
+            height: (window.innerHeight - 48 )+'px',
```
[the remark this answers is not in the document]
Comment (shayanbinary) [reply src/components/Dialog.jsx]: the `48` is the height of `Chart Control` panel. which I force dialog to be wide

[the remark this answers is not in the document]
Comment (brucebinary) [reply src/components/Dialog.jsx]: it's cleaner and clearer to have that height queried from the element itself, but if you need to put a number, at least use a descriptive variable name

File: src/components/Dialog.jsx [hunk 2]
```diff
@@ -2,31 +2,28 @@ import React, { Component } from 'react';
...
-                style={inner_style}
-                >
-                {children}
-            </div>
-        );
-    }
-}
+    return (
+        <div
+            className={(className || 'cq-dialog') + (open ? ' open' : '')}
```
Comment (brucebinary) [inline src/components/Dialog.jsx]: template literals means smth like ```{`${className || 'cq-dialog'} ${open ? 'open' : ''}`}```

File: src/components/Dialog.jsx [hunk 3]
```diff
@@ -2,24 +2,28 @@ import React, { Component } from 'react';
...
+        width: window.innerWidth+'px'
+    } : {};
 
-        return (
-            <div
-                className={(className || 'cq-dialog') + (open ? ' open' : '')}
-                onClick={onContainerClick}
+    return (
+        <div
+            className={(className || 'cq-dialog') + (open ? ' open' : '')}
```
Comment (brucebinary) [inline src/components/Dialog.jsx]: template literals means smth like ```{`${className || 'cq-dialog'} ${open ? 'open' : ''}`}```

**Pull request merged**